src/app/component/dashboard/vaucher/vaucher.component.ts

```diff
@@ -382,6 +391,10 @@ export class VaucherComponent implements OnInit {
     this.state.take = event.take;
     this.pageSize = event.take;
     this.loadProducts();
+
+    this.savePage['home/vaucher'] = event.skip;
+    this.savePage['home/vaucherTake'] = event.take;
+    this.paginationService.setLocalStorage('pageSize', this.savePage);
```

Comment:
I think it will be good to change this.
We have setLocalStorage method in helpService, so you can use that method
Or change a little bit implementation of this your method
For example:
- in paginationService you can directly make method for set/get pageSize
setGridPageSize(pageSize: any) {
localStorage.setItem('pageSize', JSON.stringify(pageSize))
}

getGridPageSize() {
return JSON.parse(localStorage.getItem('pageSize'))
}

it a more elegant and if we need to change key of pageSize localStorage value, we can do it on one plase, and get value directly